tests/certs_test.go

```diff
@@ -43,7 +43,7 @@ var _ = Describe("deis certs", func() {
 
 		Specify("that user cannot add a cert with a malformed name", func() {
 			sess, err := cmd.Start("deis certs:add %s %s %s", &user, "bogus.cert.name", cert.CertPath, cert.KeyPath)
-			Eventually(sess.Err).Should(Say("400 Bad Request"))
+			Eventually(sess.Err).Should(Say("Can only contain a-z (lowercase), 0-9 and hypens"))
```

Comment:
Hah. Just noticed that. It's actually a controller issue... the typo is in there.